### ISSUE TITLE
Predicate 'randomized' not checked for SecureRandom

### ISSUE THREAD

**Issue by SvenEV** [CROSSINGTUD/CryptoAnalysis#103]
According to the [CrySL spec for `SecureRandom`](https://github.com/CROSSINGTUD/Crypto-API-Rules/blob/master/src/de/darmstadt/tu/crossing/SecureRandom.cryptsl), the constructor and the method `setSeed(...)` require the argument to have the 'randomized' predicate. However, it seems that this requirement is not checked correctly.

In the code below, CogniCrypt does not report errors for the `SecureRandom` objects, despite the seed being hard-coded. In comparison, `KeyGenerator` and `IvParameterSpec` also have a requirement for 'randomized', and for these types errors are raised as expected.

```Java
public static void main(String[] args) throws Exception {
    byte[] seed = { 1, 2, 3 };
    
    new SecureRandom(seed); // No error 
    
    SecureRandom random = SecureRandom.getInstance("SHA1PRNG");
    random.setSeed(seed); // No error
    
    KeyGenerator keyGen = KeyGenerator.getInstance("AES");
    keyGen.init(256, random); // "Second parameter was not properly randomized"
    keyGen.generateKey();
    
    new IvParameterSpec(seed); // "First parameter was not properly randomized"
}
```

**Comment by johspaeth** [comment]
Thanks for reporting the issue. I fixed the bug and merged the fix into the master.
